### PR TITLE
InfluxDB 2.x & Cloud FAQ

### LINKED ISSUE #4029: InfluxDB Cloud FAQ
**ETA:** Already in production

The InfluxDB Cloud left nav now includes a "help bar" with links to documentation, specifically FAQ. Right now, FAQ points to the InfluxDB 1.8 FAQ. This certainly isn't ideal. We need to create 2.x and Cloud FAQ page with questions specifically for 2.x and Cloud.

### LINKED ISSUE #4032: Create task-based doc for showing Flux version
Users need a quick reference for finding the version of Flux currently installed in their instance of InfluxDB.

Notes to add:

- Release cadence differences for OSS and Cloud
- The `/api/v2/query` endpoint cannot return standalone scalar values; it only returns streams of tables. Because of this, you have to include the returned version in a stream of tables (`array.from`).
- Different ways to execute the query (API, CLI, UI)
- Note that if run from the REPL, it returns the version of Flux the REPL was built with, not the version of Flux packaged with InfluxDB.

```javascript
import "array"
import "runtime"

array.from(rows: [{version: runtime.version()}])
```

### PR DESCRIPTION
**Pull request by sanderson**
Closes #4029
Closes #4032

Adds an FAQ page for InfluxDB 2.2+ and Cloud along with new supporting documentation and style/js fixes.

- [x] Rebased/mergeable

### CODE REVIEW

**Comment by sanderson** [comment]
@blegesse-git FYI, this docs PR includes a legitimate FAQ for InfluxDB Cloud and OSS 2.2+. Once it's merged, you can update the FAQ link in the new help bar to point to `https://docs.influxdata.com/influxdb/cloud/reference/faq/` instead of the InfluxDB 1.8 FAQ.

File: content/influxdb/v2.2/reference/faq.md [hunk 1]
```diff
@@ -0,0 +1,930 @@
...
+
+## Administration
+
+{{% oss-only %}}
+
+#### How can I identify my InfluxDB version?
+
+Use one of the following methods to identify the version of InfluxDB OSS you're using:
+
+- **Look in the InfluxDB UI**:
```
Comment (sunbryely-influxdata) [inline content/influxdb/v2.2/reference/faq.md:243]: would it be better to keep this list consistent

```suggestion
- **Use the InfluxDB UI**:
```

File: content/influxdb/v2.2/reference/faq.md [hunk 2]
```diff
@@ -0,0 +1,930 @@
...
+[1.x compatiblity API](/influxdb/v2.2/reference/api/influxdb-1x/) which replicates
+the `/query` endpoint from InfluxDB 1.x. This allows all InfluxDB 1.x-compatible
+clients to work with InfluxDB {{< current-version >}}. However, InfluxQL relies
+on a database and retention policy data model doesn't exist in InfluxDB
+{{< current-version >}}, but has been replaced by [buckets](influxdb/v2.2/reference/glossary/#bucket).
+
+InfluxDB {{< current-version >}} lets you map unique database and retention 
+policy combinations used in InfluxQL to specific buckets using DBRP mappings.
+
+For details instructions on using InfluxQL with InfluxDB {{< current-version >}}
```
Comment (sunbryely-influxdata) [inline content/influxdb/v2.2/reference/faq.md:595]:
```suggestion
For detailed instructions on using InfluxQL with InfluxDB {{< current-version >}}
```

**Review by lwandzura — APPROVED** [review]
Huge amount of work, nice job!

**Review by sunbryely-influxdata — APPROVED** [review]

**Pull request merged**